CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -79,6 +79,7 @@ private static void HandleSetUp()
             _harvestApi.Get("/api/v2/time_entries")
                 .WithParameter("from", "2019-02-25")
                 .WithParameter("to", "2019-03-01")
+                .WithParameter("page", "1")
                 .Responds(harvestGetTimeEntriesResponse);
```

Comment:
`harvestGetTimeEntriesResponse` only has one page, should we paginate this?

Reply:
This is already covered in the unit tests so it is not high priority but we could revisit this later.